lib/Mojolicious/Command/prefork.pm

```diff
@@ -0,0 +1,145 @@
+package Mojolicious::Command::prefork;
+use Mojo::Base 'Mojolicious::Command';
+
+use Mojo::Server::Prefork;
+use Mojo::Util qw(getopt);
+
+has description => 'Start application with pre-forking HTTP and WebSocket server';
+has usage       => sub { shift->extract_usage };
+
+sub build_server {
+  my ($self, @args) = @_;
+
+  my $prefork = Mojo::Server::Prefork->new(app => $self->app);
+  die $self->usage
+    unless getopt \@args,
+    'a|accepts=i'            => sub { $prefork->accepts($_[1]) },
+    'b|backlog=i'            => sub { $prefork->backlog($_[1]) },
+    'c|clients=i'            => sub { $prefork->max_clients($_[1]) },
+    'G|graceful-timeout=i'   => sub { $prefork->graceful_timeout($_[1]) },
+    'I|heartbeat-interval=i' => sub { $prefork->heartbeat_interval($_[1]) },
+    'H|heartbeat-timeout=i'  => sub { $prefork->heartbeat_timeout($_[1]) },
+    'i|inactivity-timeout=i' => sub { $prefork->inactivity_timeout($_[1]) },
+    'k|keep-alive-timeout=i' => sub { $prefork->keep_alive_timeout($_[1]) },
+    'l|listen=s'             => \my @listen,
```

Comment:
Is it possible to also load the listen value from a config file like hypnotoad does?